x-pack/plugins/cases/public/components/header_page/title.tsx

```diff
@@ -28,12 +28,14 @@ const ExperimentalBadge: React.FC = () => (
 );
 
 ExperimentalBadge.displayName = 'ExperimentalBadge';
+export const TitleExperimentalBadge = React.memo(ExperimentalBadge);
 
```

Comment:
I chose to export these as individual components to also be used in `EditableTitle` (Cases) instead of removing it so I don't disturb other usages.